Bio/Blast/Applications.py

```diff
@@ -61,6 +61,7 @@ def __init__(self, cmd=None, **kwargs):
             _Option(["-outfmt", "outfmt"],
                     "Alignment view.  Integer 0-11.  Use 5 for XML output "
                     "(differs from classic BLAST which used 7 for XML).",
+                    filename=True,  # to ensure spaced inputs are quoted
```

Comment:
The docstring above needs updating to say while typically it is an integer, with 6 etc you can name columns.